tests/UnitTests/ValidatableExtensions/ValidatableExtensions.StringsTests.cs

```diff
@@ -310,4 +310,60 @@ public void ThrowIfLenghtNotEquals_WhenLengthEquals_ShouldNotThrow()
         // Assert
         action.Should().NotThrow();
     }
+
+    [TestMethod]
+    public void ThrowIfEndsWith_WhenEndsWith_ShouldNotThrow()
```

Comment:
I'm guessing you meant:
```suggestion
    public void ThrowIfEndsWith_WhenNotEndsWith_ShouldNotThrow()
```
?